ethereum_history_api/oracles/src/ethereum/client.test.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, expect, it } from 'vitest';
+import { MultiChainClient } from './client.js';
+
+describe('from_params', () => {
+  const customRpcUrl = 'http://localhost:8545';
+
+  it('setups client with custom params', () => {
+    const multiClient = MultiChainClient.from_params('mainnet', customRpcUrl);
+    const { chain, transport } = multiClient.getClient(1);
```

Comment:
Seems weird to create an object by chain name and query it by chain id in the next line. Can't we use chain id for both and resolve the name into the id first using getChainByName?

Reply:
You are right. I've ended up just preparing a client on the SDK level (pr soon)